package.json

```diff
@@ -9,11 +9,14 @@
     "index.d.ts"
   ],
   "scripts": {
-    "test": "jest"
+    "test": "jest",
+    "version": "yarn changeset version && yarn install --no-frozen-lockfile",
```

Comment:
Curious, why `yarn install` here?

Reply:
Yeah, this was copied over from the same script in the nuka-carousel repo